.github/workflows/linux-tests.yml

```
@@ -28,7 +28,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install libegl1-mesa
       - name: Install Conda
-        uses: goanpeca/setup-miniconda@v1
+        uses: conda-incubator/setup-miniconda@v2
         with:
            activate-environment: test
            auto-update-conda: true
```

We need to add here the config for the channel priority. It should be something like:
```suggestion
           auto-update-conda: true
           channels: conda-forge
           channel-priority: strict
```